dist_run.py

```diff
@@ -97,16 +109,18 @@ def main():
 
     # Distribute model on TP mesh
     model.distribute(tp_mesh)
-    logger.info(f"Model: {model}")
+    if rank == 0:
```

Comment:
minor note - I enabled log filtering by rank in the run_dist.sh launcher, thus having rank specific filtering in the code might end up being confusing. (i.e. if debugging last pp stage via global filter and then not seeing expected logging b/c we have hardcoded filtering in the code). 
~~~
LOG_RANK=${LOG_RANK:-0,1,2,3}
~~~

Reply:
Ah, okay, reverting it